modules/navier_stokes/doc/intro_slides/slides/index.md

```diff
@@ -0,0 +1,227 @@
+# The Navier-Stokes Module
+
+!style halign=center fontsize=120%
+!datetime today format=%B %Y
+
+!---
+
+# Overview
+
+Main capabilities:
+
+- Incompressible fluid flows
+- Weakly-compressible fluid flows
+- Compressible fluid flows
+- Free flows and flows in porous media describing homogenized structures
+
+Utilizies the following techniques:
+
+- Continuous Finite Element Discretization (maintained, but not intensely developed)
+- Finite Volume Discretization (current development direction)
+
+!---
+
+# The Navier-Stokes equations ([!citet](radman2021development))
```

Comment:
Why do we cite Radman et al. as a reference for the NS equations? I thought their paper was two-phase. Please consider revising

Reply:
The one that people generally cite is: K. Vafai, Handbook of Porous Media, CRC Press, Taylor & Francis, New York (2005)

Reply:
```suggestion
# The Porous Navier-Stokes equations ([!citet](radman2021development))
```

Reply:
I'll add the handbook too. I've read it but I think the formulation in Radman is the closest to what we have. It is general enough for single phase too. I'll keep them both.